HDVF/doc/HDVF/Concepts/Ring.h

```diff
@@ -0,0 +1,50 @@
+/*!
+\ingroup PkgHDVFConcepts
+\cgalConcept
+
+The concept `Ring` describes the requirements for the ring of coefficients used to compute homology in the `HDVF` concept. Besides ring operators, it also specifies the functions needed to test invertibility in the ring.
```

Comment:
I am wondering if the `Ring` concept is identical to [`IntegralDomainWithoutDivision`](https://doc.cgal.org/latest/Algebraic_foundations/classIntegralDomainWithoutDivision.html).
If it is different this concept gives an idea what you have to add to the new concept.